### PR TITLE
AG-8467 Context menu docs

### PR DESCRIPTION
**Pull request by lsjroberts**
PR for https://ag-grid.atlassian.net/browse/AG-8467

### CODE REVIEW

File: grid-packages/ag-grid-docs/documentation/doc-pages/charts-context-menu/examples/context-menu-actions/main.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+} from "ag-charts-enterprise"
+
+const options: AgCartesianChartOptions = {
+  container: document.getElementById("myChart"),
+  autoSize: true,
+  title: {
+    text: "Sweaters made",
+  },
+  contextMenu: {
+    enabled: true,
```
Comment (manapeirov) [inline grid-packages/ag-grid-docs/documentation/doc-pages/charts-context-menu/examples/context-menu-actions/main.ts:13]: do we need the enabled: true if the contextMenu obj is there

**Review by manapeirov — APPROVED** [review]
LGTM

**Review by alantreadway — APPROVED** [review]
LGTM

File: grid-packages/ag-grid-docs/documentation/doc-pages/charts-context-menu/examples/context-menu-actions/main.ts (hunk 1 above)
Comment (lsjroberts) [reply grid-packages/ag-grid-docs/documentation/doc-pages/charts-context-menu/examples/context-menu-actions/main.ts:13]: Technically not, but it's kinda rubbish to use an empty object as a boolean.

Comment (lsjroberts) [reply grid-packages/ag-grid-docs/documentation/doc-pages/charts-context-menu/examples/context-menu-actions/main.ts:13]: Oh sorry, yeah, I didn't check the full file here. I'll change it. Good shout.

**Pull request merged**